includes/footer.php

```diff
@@ -5,17 +5,83 @@
+                        <li data-target="#carouselSponsor" data-slide-to="10"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="11"></li>
+                        <li data-target="#carouselSponsor" data-slide-to="12"></li>
+                        
+                    </ol>
+
+                    <!-- Slides -->
+                    <div class="carousel-inner" role="listbox">
+                         <div class="item active">
+                            <img src="/img/sponsors/ehs.jpg" alt="/img/sponsors/ehs.jpg" title="Eastlake High School"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/google.jpg" alt="/img/sponsors/google.jpg" title="Google"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/lwsd.jpg" alt="/img/sponsors/lwsd.jpg" title="Lake Washington School District"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/boeing.jpg" alt="/img/sponsors/boeing.jpg" title="Boeing"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/chase.jpg" alt="/img/sponsors/chase.jpg" title="Chase"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/FIRSTWa.jpg" alt="/img/sponsors/FIRSTWa.jpg" title="FIRST Washington"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/godaddy.jpg" alt="/img/sponsors/godaddy.jpg" title="Go Daddy"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/honeywell.jpg" alt="/img/sponsors/honeywell.jpg" title="Honeywell"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/kenworth.jpg" alt="/img/sponsors/kenworth.jpg" title="Kenworth"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/microsoft.jpg" alt="/img/sponsors/microsoft.jpg" title="Microsoft"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/ospi.jpg" alt="/img/sponsors/ospi.jpg" title="Office of Superintendent of Public Instruction"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/rocketdyne.jpg" alt="/img/sponsors/rocketdyne.jpg" title="Aerojet Rocketdyne"/>
+                        </div>
+                        <div class="item">
+                            <img src="/img/sponsors/sae.jpg" alt="/img/sponsors/sae.jpg" title="SAE International"/>
```

Comment:
Since we present to SAE WA instead of SAE International, I think it might be better to keep the WA logo (see http://www.saenorthwest.org/?p=53).